Sources/TokamakCore/App/Scenes/Scene.swift

```diff
@@ -27,10 +27,6 @@ protocol TitledScene {
   var title: Text? { get }
 }
 
-protocol ViewContainingScene {
-  var anyContent: AnyView { get }
-}
-
```

Comment:
There doesn't seem to be a difference between `ViewContainingScene` and `SceneDeferredToRenderer`, so I'm only keeping the latter.